### ISSUE TITLE
Pixi Support

### ISSUE THREAD

**Issue by sbtq** [openforcefield/openff-utilities#87]
With the emergence of pixi (https://pixi.sh) - a new package manager to be an improvement over conda - the openff toolkit does not work anymore because the toolkit has issues to find a conda executable.

You get the following error when trying to import anything `openff.utilities.exceptions.CondaExecutableNotFoundError`.

This is due to the logic in https://github.com/openforcefield/openff-utilities/blob/main/openff/utilities/provenance.py.
Would you be fine with expanding this functionality to also check if pixi is there?

Any reasons to not support pixi?

If you want me to submit a PR please tell me, would be happy to contribute :)

**Comment by mattwthompson** [comment]
I agree `pixi` support there would be great, I keep hearing good things about it - the reason it's not there is that I haven't tried it for myself yet. We'd be happy to take a PR that expands that logic block out to handle it. It'd give me a good excuse to try it out!

**Comment by sbtq** [comment]
okay @mattwthompson, sounds good. I have seen that the ordering for those executables is important based on the comments and an issue I have seen regarding that.

Would there be anything speaking against using `pixi` as the first if found?

Will create the PR probably tomorrow then.

**Comment by sbtq** [comment]
@mattwthompson according to my tests, the changes in the referenced PR should do the trick.
Performance was also completely fine. I could not find an particular automated tests for this part of the code. WDYT?

**Comment by mattwthompson** [comment]
Thanks - will move discussion about the changes to the PR

**Comment by mattwthompson** [comment]
Not really a solution, but release 0.1.13 makes `conda list`-like failures a warning instead of an error. With #90 or something similar we might still be able to add first-class `pixi` support for this path

**Comment by sbtq** [comment]
@mattwthompson any expected timeline when the openff-toolkit is being released with this support? :)

**Comment by mattwthompson** [comment]
Strictly speaking that would come from a release of `openff-utilities`, but once that version (0.1.14?) is out, any fresh install of OpenFF tools would probably bring that down.

Anyway, probably early January - I want to include #94, which still needs a review, in the next release. And OpenFF mostly shuts down for the holidays starting next week through the new year.

**Comment by sbtq** [comment]
@mattwthompson sounds good, just wanted to understand for one of our projects that needs this feature 👍
